caravel/assets/javascripts/explorev2/reducers.js

```diff
@@ -0,0 +1,127 @@
+import shortid from 'shortid';
+import * as actions from './actions';
+
+const defaultTimeFilter = {
+  timeColumn: null,
+  timeGrain: null,
+  since: null,
+  until: null,
+};
+
+const defaultGroupBy = {
+  groupByColumn: [],
+  metrics: [],
+};
+
+const defaultSql = {
+  where: '',
+  having: '',
+};
+
+const defaultFilter = {
+  id: shortid.generate(),
+  eq: null,
+  op: 'in',
+  col: null,
+};
+
+export const initialState = {
```

Comment:
You mean load it in store?
[https://github.com/airbnb/caravel/pull/1108/files#diff-99cf1b281bb84131c9b20bfe3fb47ab7R21](url)